pkg/lib/cockpit-components-logs-panel.jsx

```diff
@@ -158,7 +158,7 @@ export class LogsPanel extends React.Component {
             <Card className="cockpit-log-panel">
                 <CardHeader>
                     <CardTitle>{this.props.title}</CardTitle>
-                    { this.props.goto_url && <CardActions><button className="link-button" role="link" onClick={e => cockpit.jump(this.props.goto_url)}>{_("All logs")}</button></CardActions>}
+                    { this.props.goto_url && <CardActions><Button variant="link" onClick={e => cockpit.jump(this.props.goto_url)}>{_("All logs")}</Button></CardActions>}
```

Comment:
This one seems misaligned. Not sure if that was the case before or not, but i think we can fix it now.
![Screenshot from 2021-04-08 10-35-18](https://user-images.githubusercontent.com/12330670/113995540-a18e5080-9856-11eb-85c9-12153eedd287.png)

Reply:
All links in card actions are misaglined https://github.com/cockpit-project/cockpit/issues/15473 

i think it would be some CSS override, which will affect so many pages, are you sure you want it part of this?

Reply:
Aha, since it is known and there is issue for it, then this is OK. Lets not fix it here.

Reply:
(it's the same in master of course)